frontend/src/v4/routes/viewerGui/components/issues/components/issueDetails/issueDetails.component.tsx

```diff
@@ -34,6 +34,7 @@ import { IssueDetailsForm } from './issueDetailsForm.component';
 interface IProps {
```

Comment:
- If I close the last risk (didnt try it with tickets or issues) it loses the position and goes straight to the top
![lost position](https://github.com/3drepo/3drepo.io/assets/42032433/b0ffa7c2-2423-4603-a7b0-085415477ff5)

Reply:
So in tickets if I close the first ticket it goes to the last ticket on the list? That's not good either. @carmenfan  what do you think?

Reply:
@ChristosTsiotsias  am I right? whatever we do it needs to be consistent between the web and the plugin

Reply:
Ok, so then ticket should work as issues

Reply:
In the plugins if the last issue/risk/custom ticket is filtered out the scroll bar goes to the top but no ticket is selected